libethcore/KeyManager.cpp

```diff
@@ -422,12 +422,20 @@ void KeyManager::write(SecureFixedHash<16> const& _key, string const& _keysFile)
 {
 	RLPStream s(4);
 	s << 1; // version
-	s.appendList(accounts().size());
+
+	s.appendList(m_keyInfo.size());
 	for (auto const& address: accounts())
```

Comment:
I think it would be better to just iterate over `m_keyInfo` - this would also better match the previous line.